web/mirage/factories/related-external-link.ts

```diff
@@ -1,11 +1,7 @@
 import { Factory } from "miragejs";
 
-// TODO: Improve how this generates IDs.
-// We should be able to use the `i` argument to generate IDs,
-// but that doesn't work when using `createList` in tests.
-
 export default Factory.extend({
-  id: 0,
+  id: (i) => i,
```

Comment:
This may have been a thing at some point but we're not currently using `createList` with `related-external-links`.